buildpipeline/Core-Setup-PortableLinux-x64.json

```diff
@@ -97,8 +97,180 @@
     },
     {
       "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Copy built Portable binaries to staging directory",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "5bfb729a-a7c8-4a78-a7c3-8d717bb7c13c",
+        "versionSpec": "2.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "SourceFolder": "$(Build.SourcesDirectory)/artifacts/linux-x64/intermediate/sharedFrameworkPublish",
+        "Contents": "**",
+        "TargetFolder": "$(Build.StagingDirectory)/sharedFrameworkPublish",
+        "CleanTargetFolder": "false",
+        "OverWrite": "false",
+        "flattenFolders": "false"
+      }
+    },
+    {
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Build Ubuntu.14.04 docker image",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "6c731c3c-3c68-459a-a5c9-bde6e6595b5b",
+        "versionSpec": "2.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "scriptPath": "scripts/dockerbuild.sh",
+        "args": "--non-interactive -d $(Build.SourcesDirectory)/scripts/docker/ubuntu.14.04 -t ubuntu.14.04-coresetup",
```

Comment:
My understanding is that we want to get rid of building docker image on the fly as part of the build step. See issue : https://github.com/dotnet/core-setup/issues/1719  . It will be easier to remove this step and consume the prebuilt images in the build step. Once issue 1719 is resolved, we can simply remove this step.